src/pages/SubmissionDetails.js

```diff
@@ -42,6 +43,24 @@ export const SubmissionDetails = ({ submissionId }) => {
       Utils.differenceFromNowInSeconds(submitted)
   }
 
+  const getFilter = filter => {
+    let filterStatement
+    switch (filter) {
+      case 'Error':
+        filterStatement = _.filter(r => errorStates.includes(r.state), runsData)
+        break
+      case 'None':
```

Comment:
if you remove this case, will it use the default case when filter is 'None'?

if switch statements work like that -- then probably best to remove this case, since it's identical to the default

Reply:
I went back and forth on this. In the list of dropdown menu options, `None` is listed, and I figured if it was passed as an input to the function, then it's fine if there isn't a case for it as it be recognized as `default`, but I wasn't sure how specific to be for the cases. I'll get rid of it however because it is returning the same data.